quantize/gptq.py

```diff
@@ -335,18 +335,15 @@ def main(
         config = Config(**json.load(fp))
 
     device = "cuda"
-    dt = getattr(torch, dtype, None)
-    if not isinstance(dt, torch.dtype):
-        raise ValueError(f"{dtype} is not a valid dtype.")
-    dtype = dt
+    fabric = Fabric(accelerator="cuda", precision=precision)
 
     # we avoid loading the entire model on the GPU and do this block by block
     checkpoint_path = checkpoint_dir / "lit_model.pth"
     print(f"Loading model {str(checkpoint_path)!r} with {config.__dict__}", file=sys.stderr)
     t0 = time.time()
-    with EmptyInitOnDevice(device="cpu", dtype=dtype):
```

Comment:
Did you check this?

Reply:
I actually overlooked this detail. Thank you for noticing.

I'm trying to run the script, but it's crashing with
```python
/tmp/tmp6xw756hu/triton/lib/Dialect/TritonGPU/Transforms/Prefetch.cpp:208: mlir::LogicalResult {anonymous}::Prefetcher::initialize(): Assertion `aKWidth == bKWidth' failed.
```

On our A100s. I tried reinstalling `pytorch-triton` and `triton` and checking out and older known good commit with no luck so far.

I'll try again on a different machine.

Reply:
Can repro on a cloud machine. Since we are installing master, maybe a triton update broke it. I'd need to check previous versions.

I'll keep this on my radar